ncm-openstack/src/main/perl/OpenStack/Client.pm

```diff
@@ -0,0 +1,73 @@
+#${PMpre} NCM::Component::OpenStack::Client${PMpost}
+
+=head1 NAME
+
+C<NCM::Component::OpenStack::Client> provides simple functions to get
+the REST client.
+
+=head1 Functions
+
+=over
+
+=cut
+
+use NCM::Component::OpenStack::Logger;
+use NCM::Component::OpenStack::Openrc;
+use Net::OpenStack::Client 0.1.2;
```

Comment:
We've published 0.1.3 in externals, can we use that version here?

Reply:
@jrha sure, done!